pkgs/by-name/re/recutils/package.nix

```diff
@@ -1,9 +1,19 @@
 { lib
 , stdenv
 , fetchurl
+, curl
```

Comment:
Why changing the order here?

Reply:
To group by dependency reason. `curl` is always required, `bc` and `check` for the check phase. I changed it to match the feature flags.

Reply:
I realize as I wrote this that the `bash` dependency doesn't follow this pattern. I can change it back if this isn't a common style.